week4/filterMore/filter/helpers.c

```diff
@@ -0,0 +1,150 @@
+#include "helpers.h"
+#include "math.h"
+
+void grayscale(int height, int width, RGBTRIPLE image[height][width])
+{
+    for (int i = 0; i < height; i ++)
+    {
+        for (int j = 0; j < width; j ++)
+        {
+            float red = image[i][j].rgbtRed;
+            float green = image[i][j].rgbtGreen;
+            float blue = image[i][j].rgbtBlue;
+            int average = round((red + green + blue) / 3);
```

Comment:
```suggestion
            int average = round((red + green + blue) / 3.0);
```
RGB成分の合計が浮動小数点数型なので、除数も浮動小数点型の値にしたいです
（3でも3.0でも処理結果に違いは無いですが、異なるデータ型で数値計算することは極力避けたいです）

Reply:
@axtx4869
除数の値を浮動小数点型に修正しました！
ご確認のほど、よろしくお願いいたします。